README.md

```diff
@@ -1,21 +1,94 @@
+ 
+Set due dates, track assignments in your teacher dashboard, or integrate other tools like testing frameworks. 
+ 
+With Classroom, you can spin up your course on GitHub and move on to the good stuff. 
+ 
+@johndbritton, @mozzadrella, @nwoodthorpe, @tarebyte, and @wilfriedE are all maintainers. 
 
-### Help wanted
+![GitHub Classroom screenshot](https://cloud.githubusercontent.com/assets/1311594/14748352/32f677b0-0887-11e6-9ac2-8aa06e4341fa.png)
 
-If you're interested in helping out with Classroom development and looking for a place to get started, check out the issues labeled [`help-wanted`](https://github.com/education/classroom/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp-wanted) and feel free to ask any questions you have before diving into the code.
+## Why try Classroom?
+ 
+*Spend more time with students, less on setup.* Students accept an assignment with one link, so you can get straight to the material. 
+ 
+*Bootstrap group assignments in a snap.* Invite students to a shared repository, and cap the number of students per group. Use the same groups over and over again, or create new ones.
+ 
+*More insight into student work than ever before.* See when students accept the assignment, and access their work from the moment they start. With version control, catch when they get stuck and help them rewind. 
+ 
+*You are in control.* Students can work individually or in groups, in public or in private. Set permissions for teaching assistants or graders. 
+ 
+*Scales for large courses with ease.* If you have a small course, Classroom will make your life easier and save you time. f you have hundreds of students, we have your covered: as many repositories as you need, and webhooks to integrate automated testing tools. 
+ 
+*Works with your Learning Management System (LMS).* Students submit a link to their assignment repository to your learning management system. Give feedback through comments in GitHub, but keep grades in your LMS.
+ 
+*You’re not alone.* The experts on the GitHub Education team are here to answer any of your questions, and we’ve got docs, best practices, and a strong community of educators to help you migrate to Classroom.
+ 
+*Are you super-advanced?* Do you want to build your own tools? We 💖 contributions. 
+
+## Design principles
+
+*Classroom is a teacher-facing tool to enable the educational use of GitHub.* Every student needs feedback and support as they learn to code, and using GitHub you can give them the right advice, in the right place, at the right time. This tool makes it easier for you to use the workflow you love, but with some organization, automation and ease with your students. 
+ 
+*Students use GitHub. They don’t use Classroom.* Experience with real-world tools gives students a leg-up once they move on from school. Invest time in the tools students can grow with, not another third-party tool with its own learning curve. 
```

Comment:
Technically GitHub is a third-party tool with it's own learning curve, maybe this could be re-worded to explain how experience with GitHub will be applicable in the real-world / industry